lib/airbrake/shared_tasks.rb

```diff
@@ -18,6 +18,7 @@
                          :scm_repository => ENV['REPO'],
                          :local_username => ENV['USER'],
                          :api_key        => ENV['API_KEY'],
+                         :message        => ENV['DEPLOY_MESSAGE'],
```

Comment:
We don't support this deploy option so I'm not sure how this could improve/fix anything.